custom_components/ha_washdata/profile_store.py

```
@@ -6239,6 +6211,18 @@ def get_points(cy: CycleDict) -> list[tuple[float, float, float]]:
 
+        # Recompute energy from the merged trace (merge keeps only c1's original
+        # energy_wh which under-represents the combined cycle).
+        try:
+            ts_s = np.array([pt[0] for pt in new_power_data], dtype=float)
+            p_s = np.array([pt[1] for pt in new_power_data], dtype=float)
+            if len(ts_s) > 1:
+                c1["energy_wh"] = round(
+                    integrate_wh(ts_s, p_s, max_gap_s=energy_gap_threshold_s(ts_s)), 3
+                )
+        except Exception:  # pylint: disable=broad-exception-caught
+            pass  # stale energy_wh is better than a broken merge
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)